flipperdevices/Flipper-Android-App
# Fix wearos Dispatchers.Default

**Background**

Currently wearOS application has a bug, which is looks like problem with connectvitiy. It happened because a lot of coroutines launched in `Dispatchers.Default` as well as DecomposeViewModels.

**Changes**

- Fixed WearEmulateViewModel getWearEmulateState combine via adding custom `FlipperDispatchers.workStealingDispatcher`

**Test plan**

- Open wearOS application 
- Select remote
- Open it and see that state on the screen now is changed and yout don't need to reload it multiple times
